ansible_collections/arista/avd/roles/eos_designs/doc/custom-structured-configuration.md

```diff
@@ -275,7 +269,7 @@ custom_structured_configuration_ethernet_interfaces:
     peer_type: my_precious
 ```
 
-In this example the contents of the `name_server.nodes` variable in the Structured Configuration will be replaced by the list `[ 10.2.3.4 ]`
+In this example the contents of the `ip_name_servers` variable in the Structured Configuration will be replaced by the list `[ 10.2.3.4 ]`
```

Comment:
```suggestion
In this example the contents of the `ip_name_servers` variable in the Structured Configuration will be replaced by the list `[{"ip_address": "10.2.3.4", "vrf": "MGMT"}]`
```

Reply:
But actually all this needs to be rewritten. The new default behavior is the first schema merge on primary key, and secondly use append_rp

Reply:
hmmm .. maybe let's take it in a different PR?

Reply:
yeah

Reply:
https://github.com/aristanetworks/ansible-avd/issues/2742